mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -357,80 +384,147 @@ object OptionGroup {
     } yield Typechecker(inferpoly)
   }
 
+  /** Options used to rack specification predicates */
+  case class Predicates(
+      behaviorSpec: BehaviorSpec,
+      cinit: Option[String],
+      invariants: List[String],
+      temporal: List[String],
+      tlcConfig: Option[(TlcConfig, File)],
+      view: Option[String])
+      extends OptionGroup
+
+  object Predicates extends Configurable[Config.Checker, Predicates] {
```

Comment:
I think this needs a bit of doc on how it handles TLC configs.
Found it hard to follow based on the code alone.

Reply:
Good call! Most likely I will be the one most helped by this in the immediate future :D 

See 66951c6d935270f31abbe3242b1e8e51ad9d8f3f